bears/yaml/TravisLintBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+
+
+@linter(executable='travis',
+        output_format='regex',
+        output_regex=r'\[x\]\s+(?P<message>.+)')
```

Comment:
Yes, even I did not like it at first, but there is no better alternative. See https://github.com/coala/coala-bears/issues/294#issuecomment-304322328